src/root/WithdrawManager.ts

```diff
@@ -169,61 +172,51 @@ export default class WithdrawManager extends ContractsBase {
     return this.web3Client.send(txObject, _options)
   }
 
-  async startExitForMintWithTokenURITokens(burnTxHash, options?) {
-    const { payload, mint } = await this._startExitForMintWithTokenURITokens(
-      burnTxHash,
-    )
-    return this.web3Client.send(
-      this.erc721Predicate.methods.startExitForMintWithTokenURITokens(
-        payload,
-        mint,
-      ),
-      options,
-    )
+  /**
+   * Start an exit for a token with metadata (token uri) that was minted and burnt on the side chain
+   * Wrapper over contract call: [MintableERC721Predicate.startExitForMintableBurntToken](https://github.com/maticnetwork/contracts/blob/e2cb462b8487921463090b0bdcdd7433db14252b/contracts/root/predicates/MintableERC721Predicate.sol#L31)
+   * @param burnTxHash Hash of the burn transaction on Matic
+   * @param predicate address of MintableERC721Predicate
+   */
+  async startExitForMintableBurntToken(burnTxHash, predicate: address, options?) {
```

Comment:
any specific reason to pass `predicate address` as a parameter,
can't we directly use from global variable `erc721Predicate: Contract`

Reply:
Discussed offline.